Source/WebKit/NetworkProcess/storage/LocalStorageManager.cpp

```diff
@@ -185,7 +185,7 @@ void LocalStorageManager::connectionClosedForTransientStorageArea(IPC::Connectio
     m_transientStorageArea = nullptr;
 }
 
-StorageAreaIdentifier LocalStorageManager::connectToLocalStorageArea(IPC::Connection::UniqueID connection, StorageAreaMapIdentifier sourceIdentifier, const WebCore::ClientOrigin& origin, Ref<WorkQueue>&& workQueue)
+StorageAreaBase* LocalStorageManager::ensureLocalStorageArea(const WebCore::ClientOrigin& origin, Ref<WorkQueue>&& workQueue)
```

Comment:
Why is this returning a raw pointer and not a C++ reference? Seems like it can never return null?